Classes/Command/InstallCommandController.php

```diff
@@ -115,22 +117,23 @@ public function generatePackageStatesCommand($removeInactive = false, $activateD
                 if (empty($activePackages[$package->getPackageKey()])) {
                     $this->packageManager->unregisterPackage($package);
                     GeneralUtility::flushDirectory($package->getPackagePath());
-                    $this->outputLine('Removed Package: ' . $package->getPackageKey());
+                    $this->outputLine('Removed Package: '.$package->getPackageKey());
```

Comment:
This violates the code style. There should even be a fixer defined in .php_cs that adds the spaces